networking/network_observability/installing-operators.adoc

```diff
@@ -1,13 +1,17 @@
 :_content-type: ASSEMBLY
 [id="installing-network-observability-operators"]
-= Installing Network Observability operators in {product-title}
+= Installing the Network Observability operator
 include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
+The Loki Operator integrates with the Network Observability operator to implement multi-tenancy & authentication. Installing the Loki operator is a  prerequisite for using the Network Observability operator. You can install the the Loki Operator using the {product-title} web console Operator Hub. 
 
-The External DNS Operator deploys and manages `ExternalDNS` to provide the name resolution for services and routes from the external DNS provider to {product-title}.
+[IMPORTANT]
+====
+The Network Observability operator is not multi-tenant. You must provide a dedicated LokiStack instance for it.
```

Comment:
Will this make sense to someone who isn't all the familiar with loki or even multi-tenancy?  
We might want to put some more context around what this means.

Reply:
@davegord I updated this to add more context around Loki, LokiStack and hopefully multi-tenancy within those connotations..borrowed a lot from the Logging Loki/LokiStack explanation, so hoping it all hangs here as well.